torch/_inductor/lowering.py

```diff
@@ -3550,6 +3552,42 @@ def fn(idx):
     )
 
 
+@register_lowering(quantized.max_pool2d, type_promotion_kind=None)
+def quantized_maxpool2d(
```

Comment:
how many more lowerings like this will we have ? Could we add `torch/_inductor/quantized_lowerings.py` ?

Reply:
Hi @eellison, Thanks for the comments. I will move it to `torch/_inductor/quantized_lowerings.py`.

- In general, I think we should reply on the cpp backend auto vectorization code generation to get good performance. Then, we can avoid unnecessary `ExternKernelAlloc`. I will explore the solution to get a good performance.
- However, in short term and before a good performance solution of auto vectorization code generation with uint8 data type, we will add some `ExternKernelAlloc` to `torch/_inductor/quantized_lowerings.py` for quantization use case.